### DISCUSSION TITLE
咨询大家个问题，我怎么感觉模板打印的宋体小四1.5倍行距，比word这么设置的，显得更大更宽呢？

### DISCUSSION THREAD

**Discussion by sastars9** [nju-lug/NJUThesis#198] [Q&A]
是因为页边距设置大小的问题吗？

**Comment by note286** [comment]
请证明你的感觉。

**Reply by sastars9** [reply]
我是看到不同打印版本，仔细核对之后感觉确实是页边距设置
![image](https://user-images.githubusercontent.com/128053942/228531846-a6806130-8914-40b7-8447-dd2b988d6f7c.png)
左边是模板导出，右边是word转pdf
